tests/files/packet_centos7-flannel-containerd-addons-ha.yml

```diff
@@ -38,3 +38,19 @@ tls_cipher_suites:
 etcd_tls_cipher_suites:
   - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
   - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+
+# test MetalLB
+metallb_enabled: true
+metallb_ip_range:
+  - "10.5.0.50-10.5.0.99"
```

Comment:
How to specify this IP range?
Can we allocate this range always at CI?

Reply:
This is just to specify the ip range, it will not be allocated in the cluster.

Reply:
That means we can test the MetalLB deployment itself but cannot test the functionality of the deployed MetalLB like accessing pods via LB service, right?